cdap-ui/app/cdap/components/CaskWizards/UploadData/index.js

```diff
@@ -50,10 +50,18 @@ export default class UploadDataWizard extends Component {
     let fileContents = state.viewdata.data;
     let currentNamespace = NamespaceStore.getState().selectedNamespace;
     let authToken = cookie.load('CDAP_Auth_Token');
-    this.buildSuccessInfo(packagename, streamId, currentNamespace);
+    // currently no usecases so far require a CTA for this step
+    if (!this.props.isUsecase) {
```

Comment:
We should have a wrapper just like we do for other cases and use that wrapper for usecases rather than directly this one. We should have the `successInfo` prop being consistently used everywhere